### ISSUE TITLE
Lesson 8: TensorFlow - capitalization: "TensorFLow"

### ISSUE THREAD

**Issue by whyboris** [udacity/dlnd-issue-reports#604]
TensorFlow lesson 8 has incorrectly capitalized "TensorFlow" in the last sentence of the lesson:
"In the upcoming TensorFLow Lab, you'll get the opportunity to choose your own learning rate, epoch count, and batch size to improve the model's accuracy."
